site/static/examples/jobs/sample-scalable-job.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sample-job-
+  namespace: default
+  annotations:
+    kueue.x-k8s.io/elastic-job: "true"
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 3
```

Comment:
Don't we need to set completionMode?

I think NonIndexedJobs are immutable and you would have to have an IndexedJob to actually allow for scaling of a Job.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/batch/validation/validation.go#L860